modules/store-devtools/src/config.ts

```diff
@@ -139,6 +149,8 @@ export function createConfig(
     serialize: false,
     logOnly: false,
     autoPause: false,
+    trace: false,
+    traceLimit: 10,
```

Comment:
We should increase this.
The default of `10` doesn't work here this context (Angular + RxJS), because we have larger stacktraces compared to react.
If we don't do this, the trace window doesn't show us the trigger source, or we have to set the trace manually in every app.

```suggestion
    traceLimit: 75,
```

An alternative would be to override the trace implementation, and trim a few traces from the stacktrace.
This isn't needed imho, but could be useful for in the future.

```ts
trace: () => {
        Error.stackTraceLimit = 10000;
        const  trace = Error().stack.split('\n').slice(40, 80).join('\n')
        Error.stackTraceLimit = 10;
        return trace;
}
```